browser-interface/packages/shared/profiles/sagas/comms/index.ts

```diff
@@ -23,11 +23,12 @@ export async function fetchPeerProfile(
         return state.profiles.userInfo[userId]!.data
       }
     }),
-    delay(COMMS_PROFILE_TIMEOUT)
+    sleep(COMMS_PROFILE_TIMEOUT).then(() => null)
```

Comment:
```suggestion
    sleep(COMMS_PROFILE_TIMEOUT)
```
No need to return it!